src/engine/N3Base/N3FXBundle.cpp

```diff
@@ -457,6 +457,15 @@ bool CN3FXBundle::Load(HANDLE hFile) {
         ReadFile(hFile, &m_bStatic, sizeof(bool), &dwRWC, NULL);
     }
 
+    if (m_iVersion == 3) {
```

Comment:
I don't know if there's any real intention of supporting them, but I would personally make this `m_iVersion >= 3` to allow for future versions since there's no real cost for doing so, and only unintentional complications down the track as a result of the explicit check.